extensions/wikia/NotAValidWikia/NotAValidWikiaHooks.class.php

```diff
@@ -10,7 +10,8 @@ public static function onArticleFromTitle( &$title, &$article ) {
 		global $wgNotAValidWikia;
 
 		// http://community.wikia.com/wiki/Community_Central:Not_a_valid_community -> Community_Central:Not_a_valid_community
-		$notAValidWikiaArticleName = array_pop( explode( '/', $wgNotAValidWikia ) );
+		$notAValidWikiaParts = explode( '/', $wgNotAValidWikia );  // assign explode result to a var as array_pop requires a reference
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app